packages/graphql/src/lib/questions.ts

```diff
@@ -0,0 +1,65 @@
+import { Question, QuestionType } from '@klicker-uzh/prisma'
+import * as R from 'ramda'
+import {
+  AllQuestionTypeData,
+  ChoicesQuestionData,
+  FreeTextQuestionData,
+  NumericalQuestionData,
+  QuestionResults,
+  QuestionResultsChoices,
+} from 'src/types/app'
+
+export function processQuestionData(question: Question) {
+  const extractRelevantKeys = R.pick([
```

Comment:
Could we extract these keys from a GraphQL type for questionData for example using Object.keys(xy) in order to avoid hard-coded parameters?

Reply:
good idea & done :)